ci/environment-docs.yaml

```diff
@@ -21,7 +21,7 @@ dependencies:
   - psutil
   - python=3.8
   - sortedcontainers
-  - scikit-learn>=1.0.0
+  - scikit-learn<=1.1.*
```

Comment:
This version constraint is different than the `scikit-learn <=1.1.3` one used elsewhere. Just wanted to double check this is intentional

Reply:
Nope! Thanks